arcor2_AREditor/Assets/BASE/Scripts/Parameter.cs

```diff
@@ -609,7 +633,10 @@ public class Parameter : IO.Swagger.Model.ActionParameter {
                     parameter = InitializeRelativePoseParameter(actionParameterMetadata, handler, Parameter.GetValue<IO.Swagger.Model.Pose>(value), linkable);
                     break;
                 case "pose":
-                    parameter = InitializePoseParameter(actionParameterMetadata, layoutGroupToBeDisabled, canvasRoot, handler, value, type, linkable, windowToHideWhenRequestingObj);
+                    parameter = InitializePoseParameter(actionParameterMetadata, layoutGroupToBeDisabled, canvasRoot, handler, value, type, linkable, windowToHideWhenRequestingObj, actionPoint);
+                    break;
+                case "position":
```

Comment:
Why not use `case POSITION:` as in `ParameterMetadata.cs`?

Reply:
good point, changed at other places as well